src/styles/_typography.scss

```diff
@@ -15,10 +19,21 @@ a[href] {
     text-decoration: none;
     outline: var(--gcds-border-width-md) solid var(--gcds-focus-background);
   }
-}
 
-a[href]:not([class*="link-inherit"]):not([class*="link-light"]):not(:focus):not(:hover) {
-  color: var(--gcds-link-default);
+  // Exceptions
+  .text-light & {
+    color: currentColor;
+    &:hover {
+      color: currentColor;
+    }
+  }
```

Comment:
Why would we write additional CSS if we can target the exceptions in one line above?